docs/book/src/topics/ssh-access.md

```diff
@@ -112,6 +112,27 @@ spec:
 If you specify a security group to be associated with the Azure Bastion subnet, it needs to have some networking rules defined or
 the `Azure Bastion` resource creation will fail. Please refer to [the documentation](https://docs.microsoft.com/en-us/azure/bastion/bastion-nsg) for more details.
 
+There are also two additional options for the `Azure Bastion` resource:
```

Comment:
rather than adding a separate doc section for these two options, what do you thing of adding them into the existing example above?